app/views/users/contributors.html.erb

```diff
@@ -23,7 +23,7 @@
     <% @users.each do |user| %>
     <tr>
       <td><% if user.spectrum_count > 0 %><img width="120px" class="img" src="<%= user.spectrums.order('id DESC').limit(1).last.photo.url(:thumb) %>" /><% end %></td>
-      <td><a href="<%= profile_path(user.login) %>"><%= user.login %></a> 
+      <td><a href="/profile/<%= user.login %>"><%= user.login %></a> 
```

Comment:
Apparently now profile_path(login) gives /profile.login instead of /profile/login